lib/SGN/Controller/AJAX/BrAPI.pm

```diff
@@ -2609,7 +2637,7 @@ sub observation_units_POST {
 	my $self = shift;
 	my $c = shift;
 	# The observation units need an operator, so login required
-	my $force_authenticate = 1;
+	my $force_authenticate = 0;
```

Comment:
Add a config to require auth or not

Reply:
makes sense but force authentication may be required by specific endpoints.
force_authenticate value  still should remain as 1